src/nexpy/gui/pyqt.py

```diff
@@ -1,8 +1,10 @@
+import matplotlib
 import sip
 for api in ['QString', 'QVariant']:
     sip.setapi(api, 2)
 
-from qtconsole.qt import QtCore, QtGui, QtSvg, QT_API
+matplotlib.use('Qt4Agg')
```

Comment:
Since I have basically decided to stick to PyQt4 for now, that is also an option. I had hoped that MPL would be a better route to PyQt5 compatibility, but I have other issues with PyQt5 on my Mac.